### PR TITLE
Remove parents from dependents page

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3492
screen shot 2017-06-29 at 4 14 40 pm

### CODE REVIEW

File: src/js/pensions/config/form.js [hunk 1]
```diff
@@ -99,10 +98,6 @@ function isCurrentMarriage(form, index) {
   return isMarried(form) && numMarriages - 1 === index;
 }
 
-function isChild(item) {
-  return item.dependentRelationship === 'child';
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Will `dependentRelationship` ever not be child now?

**Review by cehsu — APPROVED** [review]

File: src/js/pensions/config/form.js [hunk 2]
```diff
@@ -853,9 +830,8 @@ const formConfig = {
                 minItems: 1,
                 items: {
                   type: 'object',
-                  required: ['dependentRelationship', 'fullName'],
+                  required: ['fullName', 'childDateOfBirth'],
                   properties: {
-                    dependentRelationship: dependents.items.properties.dependentRelationship,
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Are there any changes that should be made to the back end schema to reflect this?

Comment (jbalboni) [inline src/js/pensions/config/form.js]: Yep, I made a change to remove this field in the backend schema.

File: src/js/pensions/config/form.js [hunk 3]
```diff
@@ -853,9 +830,8 @@ const formConfig = {
                 minItems: 1,
                 items: {
                   type: 'object',
-                  required: ['dependentRelationship', 'fullName'],
+                  required: ['fullName', 'childDateOfBirth'],
```
Comment (cvalarida) [inline src/js/pensions/config/form.js]: Does adding the `childDateOfBirth` here make it required even when there are no depended children to report?

Comment (jbalboni) [inline src/js/pensions/config/form.js]: No, they won't see this array/page at all and won't hit validation.

**Pull request merged**